modules/lambda-function/main.tf

```diff
@@ -24,7 +24,11 @@ resource "aws_lambda_function" "compute_queue_backlog" {
 
   source_code_hash = filebase64sha256(data.archive_file.compute_queue_backlog.output_path)
 
-  layers = ["arn:aws:lambda:${data.aws_region.current.name}:${local.datadog.account_id}:layer:Datadog-${local.datadog.layer_runtime}:${local.datadog.layer_version}"]
+  layers = concat(
+    ["arn:aws:lambda:${data.aws_region.current.name}:${local.datadog.account_id}:layer:Datadog-${local.datadog.layer_runtime}:${local.datadog.layer_version}"],
```

Comment:
Should `Datadog` be capitalized? Not sure about the layer naming standard.

Reply:
It has to be, as otherwise it won't be found (layer refs are case sensitive, as they are just arns). Not sure if there is a naming standard for lambda layers. I did not see a 'best practices' section in the documentation, though I was not looking for one.